Classes/Service/Menu/MenuGenerator.php

```
@@ -0,0 +1,139 @@
+        $this->menuButtonBuilder->addEditSection($menuButton, $contentElement, $languageUid, $pid, $returnUrlAnchor);
+        $this->menuButtonBuilder->addActionSection($menuButton, $contentElement, $returnUrlAnchor);
+
+        return $menuButton;
+    }
+
+    private function handleAdditionalData(
+        Button $button,
+        array $contentElement,
+        array $contentElementConfig,
+        array $data,
+        BackendUserAuthentication $backendUser,
```

PHPMD (2.15.0)

Avoid unused parameters such as '$backendUser'. (Unused Code Rules)

(UnusedFormalParameter)